web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -636,6 +654,97 @@ export function AppAccessSpecSection({
   );
 }
 
+export function DatabaseAccessSpecSection({
+  value,
+  isProcessing,
+  onChange,
+}: SectionProps<DatabaseAccessSpec>) {
+  return (
+    <>
+      <Box mb={3}>
+        <Text typography="body3" mb={1}>
+          Labels
+        </Text>
+        <LabelsInput
+          disableBtns={isProcessing}
+          labels={value.labels}
+          setLabels={labels => onChange?.({ ...value, labels })}
+        />
+      </Box>
+      <FieldSelectCreatable
```

Comment:
That's a good point. I'll disable these, just as we disable them in the user invitation dialog.

Reply:
@ravicious Well, I thought it's an one-liner, but no. The existing UI that does it better way actually reimplements a part of the internal React Select logic; basically takes over the responsibility of creating items. Since this is not a reusable component, I'd rather first discuss what we would like the interaction to look with the UX team and then apply it consistently to other places (yes, we do the same thing in other places).

Reply:
Plot twist! I found out about `openMenuOnClick`, I'll use that. It was a one-liner, after all. Well, almost.